stock_dashboard_api/views/stock_view.py

```diff
@@ -17,13 +19,28 @@ class StockView(MethodView):
     """
 
     def get(self, pk: int) -> Response:  # pylint: disable=C0103, R0201
-        """A method that return Stock if provided pk is valid
+        """A method that return Stock if provided pk is valid, or if provided valid pk, 'from' and 'to' in query string,
+         this method return list of stock data for some period of time.
 
         :param pk: Stock primary key (id)
         :return: Response with one Stock
         """
         if isinstance(pk, int):
             stock = Stock.get_by_id(pk)
+            datetime_from, datetime_to = request.args.get('from'), request.args.get('to')
+            if stock and datetime_from and datetime_to:
+                try:
+                    datetime_from = datetime.strptime(datetime_from, DATETIME_PATTERN)
+                    datetime_to = datetime.strptime(datetime_to, DATETIME_PATTERN)
+                except ValueError:
+                    return make_response(
+                        "Incorrect date specified, example '18/09/19 01:55:19'(year/month,day hour:minute:second)",
+                        400)
+
+                stock_data_for_time_period = stock.get_data_for_time_period(datetime_from, datetime_to)
+                stock_data_for_time_period = convert_stock_data_to_dict_per_time_period(stock_data_for_time_period)
```

Comment:
[d.to_dict() for d in stock_data_per_time_period]

Reply:
Omg, `convert_stock_data_to_dict_per_time_period` - its length is 42